### DISCUSSION TITLE
statt ° wird Ȃ° angezeigt im einem eigenen eCharts

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2522] [General]
So soll es sein
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/4e96d7e9-08b3-4f1d-9798-b1e75d36ef6c)

So wird es bei einem anderen angezeigt
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/77fea283-974e-4903-8146-9b182f81742d)

UTF-8 ist gesetzt
fontFamily für eCharts in den options ist gesetzt
` fontFamily: "Arial,sans-serif",`

Hast du eine Idee was man da noch machen kann?

**Comment by Zefau** [comment]
Du hast einfach `°C` als Einheit angegeben und jarvis fügt da `Å` hinzu?

**Comment by mcuiobroker** [comment]
Nein, kein jarvis-Standard. Das ist ein Zusatzprogramm. In jarvis scripts geschrieben. Ich dachte das wäre bei dir schon mal vorgekommen.
```
html += '                   fontFamily: "Arial,sans-serif",'; 
html += '                   formatter: "{value}°C"';
```

**Comment by mcuiobroker** [comment]
Man muss dann Unicode verwenden \u00B0C
